src/WebUI/Components/DaemonSetListingComponent.tsx

```diff
@@ -28,7 +28,9 @@ export class DaemonSetListingComponent extends BaseComponent<IDaemonSetComponent
             <div>{
                 <ListComponent
                     className={css("list-content", "top-padding", "depth-16" )}
-                    items={this.props.daemonSetList.items || [] }
+                    items={(this.props.daemonSetList.items || []).filter((item)=>{
```

Comment:
for formatting there are few options available you can use them

"editor.formatOnType": true,

etc

so that we can have a formatterd code, this is just to be better readability nothing else